vars/main.yml

```diff
@@ -1,3 +1,13 @@
 ---
 # vars file for gotop
-gotop_download_url: https://github.com/cjbassi/gotop/releases/download/{{ gotop_version }}/gotop_{{ gotop_version }}_linux_amd64.tgz
+
+go_arch_map:
+  i386: '386'
+  x86_64: 'amd64'
+  aarch64: 'arm64'
+  armv7l: 'arm7'
+  armv6l: 'arm6'
+
+go_arch: "{{ go_arch_map[ansible_architecture] | default(ansible_architecture) }}"
+
+gotop_download_url: https://github.com/xxxserxxx/gotop/releases/download/v{{ gotop_version }}/gotop_v{{ gotop_version }}_{{ ansible_system | lower }}_{{ go_arch }}.tgz
```

Comment:
Hey, I noticed that user difference, was: `cjbassi` and proposed: `xxxserxxx`.  When I google `gotop`, I see user `cjbassi` show up.

Is there a good reason to change the repository? Maybe it should be a user selectable value...

Reply:
Hey,

yes, `cjbassi` no longer support it's `gotop` version (see disclaimer here: https://github.com/cjbassi/gotop)

`xxxserxxx` is the """official""" fork (also available in the disclaimer)